custom/icds_reports/reports/bihar_api.py

```diff
@@ -120,7 +120,11 @@ def get_mother_details(month, state_id, last_ccs_case_id):
         'anc_1',
         'anc_2',
         'anc_3',
-        'anc_4'
+        'anc_4',
+        'total_ifa_tablets_received',
```

Comment:
I think you missed to add these fields to be added to `BiharAPIMotherView` model.

Reply:
updated